test/cpp/interop/stress_test.cc

```diff
@@ -202,7 +202,7 @@ int main(int argc, char** argv) {
 
   gpr_log(GPR_INFO, "Starting test(s)..");
 
-  vector<thread> test_threads;
+  vector<std::thread> test_threads;
```

Comment:
Can you change this to grpc::thread? (I have a pending review where I changed it to grpc::thread. https://github.com/grpc/grpc/pull/4035/files#diff-c9e122f7d296d0b8ec73812d1f7f459f).  

Btw, you will need "#include <grpc++/impl/thd.h>" header for this.